server/service.go

```diff
@@ -1335,16 +1153,10 @@ func (m *BoostService) handleGetPayload(w http.ResponseWriter, req *http.Request
 	// Decode the body now
 	payload := new(eth2ApiV1Deneb.SignedBlindedBeaconBlock)
 	if err := DecodeJSON(bytes.NewReader(body), payload); err != nil {
-		log.Debug("could not decode Deneb request payload, attempting to decode body into Capella payload")
-		payload := new(eth2ApiV1Capella.SignedBlindedBeaconBlock)
```

Comment:
why are we skipping this now? Are we dropping Capella support altogether?

Reply:
I started to remove some part of Capella support to address the TODO in #9. Is there any reason to keep Capella code now?

Reply:
I don't mind it, just wanted to understand why you did it

Reply:
Just dropping unused code or useless (now) edge cases. We'll never accept Capella bids so we can safely remove everything related to it. The less code there is in this fork, the better!